apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -129,7 +129,9 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
                 iec: get(recipient, 'iec'),
                 bic: get(recipient, 'bic'),
                 bankAccount: get(recipient, 'bankAccount'),
+                deletedAt: null, // TODO(zuch): DOMA-2395 Move deletedAt filter to getOne
```

Comment:
It should be passed automatically I guess, you should add deletedAt filter only to native keystone request…to something like getByCondition…and how about getByCondition here?

Reply:
getOne uses getAll and getAll uses execGqlWithoutAccess
I added todo to move this filter to getOne function